examples/albert/README.md

```diff
@@ -56,8 +56,8 @@ To join the collaboration with a GPU trainer,
 - Run:
   ```bash
   python run_trainer.py \
-  --experiment_prefix SAME_AS_IN_RUN_TRAINING_MONITOR --initial_peers ONE_OR_MORE_PEERS --seed 42 \
-  --logging_first_step --logging_steps 100  --output_dir ./outputs --overwrite_output_dir --logging_dir ./logs
+  --experiment_prefix my-albert-v1 --initial_peers ONE_OR_MORE_PEERS \
```

Comment:
Also, I replace `SAME_AS_IN_RUN_TRAINING_MONITOR` with `my-albert-v1`, so a reader does not have to look up the default experiment prefix in the previous section of the readme.